src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -39,8 +44,11 @@
 
     @Test
     public void getStatisticAboutOrange() {
+        try {
         workWithFile.getStatistic("orange.csv", ORANGE_RESULT_FILE);
-
+        } catch (FileNotFoundException e) {
+            System.out.println("Can't write data to file");
```

Comment:
The error message 'Can't write data to file' is not very informative. Consider using a more descriptive message that includes the exception details.